scripts/uart_terminal/uart_terminal.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+import sys
```

Comment:
Can all these scripts be combined? A user may think they can run any of the scripts in the scrips folder, but those may be better off as private class members.

Reply:
I am adding the __name__ as commented below. This will indicate that this is the main and the others are included into it, also it is the only script mentioned in the usage section of the document.